mlflow/pyfunc/__init__.py

```diff
@@ -421,7 +421,6 @@ def predict(self, context, model_input, params=None):
 import numpy as np
 import pandas
 import yaml
-from databricks.sdk import WorkspaceClient
```

Comment:
Interesting. Why this doesn't work. `databricks-sdk` is a dependency.

Reply:
Seems like `Recipe(profile="databricks")` introduced circular imports on databricks.sdk